### PR TITLE
Bump cxf version to 3.3.5 from 3.3.2

### PR DESCRIPTION
**Pull request by aledsage**
(to match Karaf 4.2.8)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -67,11 +67,11 @@
     <mockwebserver.version>3.5.0</mockwebserver.version>
 
     <!-- Production -->
-    <cxf.version>3.3.2</cxf.version>
+    <cxf.version>3.3.5</cxf.version>
     <cxf.java2ws.plugin.version>${cxf.version}</cxf.java2ws.plugin.version>
-    <httpcomponents.httpcore.version>4.4.11</httpcomponents.httpcore.version>
+    <httpcomponents.httpcore.version>4.4.12</httpcomponents.httpcore.version>
```
Comment (nakomis) [inline pom.xml:72]: https://github.com/apache/brooklyn-server/pull/1096/files is showing 4.4.13

File: pom.xml [hunk 2]
```diff
@@ -67,11 +67,11 @@
...
     <httpcomponents.httpasyncclient.version>4.1.4</httpcomponents.httpasyncclient.version>
-    <httpcomponents.httpclient.version>4.5.8</httpcomponents.httpclient.version>
+    <httpcomponents.httpclient.version>4.5.10</httpcomponents.httpclient.version>
```
Comment (nakomis) [inline pom.xml:74]: https://github.com/apache/brooklyn-server/pull/1096/files is showing 4.5.12

File: pom.xml (hunk 1 above)
Comment (aledsage) [reply pom.xml:72]: 4..4.12 is right - I looked in cxf 3.3.5 feature definition in an installed karaf with `feature:info`.

File: pom.xml (hunk 2 above)
Comment (aledsage) [reply pom.xml:74]: As above: 4.5.10 is right.

**Pull request merged**